### DISCUSSION TITLE
CVE-2023-22809 vulnerability

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#9879] [General] [answered]
Hello all,

My security onion with SO Custom CENTOS is up to date. I installed latest patches yesterday for both SO and OS.

Is it safe against that sudo/sudoedit vulnerability (CVE-2023-22809)?
How can I check it since standard CENTOS commands are not working?

Thank you.
Isac

**Comment by TOoSmOotH** [comment, marked as answer]
https://lists.centos.org/pipermail/centos-announce/2023-January/086363.html

According to the errata advisory above this issue is patched in sudo-1.8.23-10.el7_9.3.x86_64.

From a machine I applied the hotfix to yesterday:

$ rpm -qa | grep sudo
sudo-1.8.23-10.el7_9.3.x86_64

I have no idea what you mean by standard commands not working.

**Reply by IzacFrank** [reply]
Thank you so much.
